pilot/pkg/security/authz/model/model.go

```diff
@@ -68,6 +68,7 @@ const (
 	// Envoy config attributes for ServiceRole rules.
 	methodHeader = ":method"
 	pathHeader   = ":path"
```

Comment:
Does this still needed, if so, in what cases? Maybe add some comment to be clear.

Reply:
cleaned other code as well.